.github/workflows/master.yml

```
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+name: Master workflow
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Build
+        run: |
+          CGO_ENABLED=1 go build -o build/_output/onos-uenib ./cmd/onos-uenib
```

Do you need `CGO_ENABLED=1`?
```suggestion
        run: CGO_ENABLED=1 go build -o build/_output/onos-uenib ./cmd/onos-uenib
```